modules/bookclass.js

```diff
@@ -0,0 +1,83 @@
+import { hideSection, listSection } from './eventlisteners.js';
+
+export const title = document.getElementById('title');
+export const author = document.getElementById('author');
+export const error = document.getElementById('error');
+
+export class BookList {
+  constructor() {
+    this.myData = this.getStorage();
+  }
+
+    setStorage = () => localStorage.setItem('bookData', JSON.stringify(this.myData)) || [];
+
+    getStorage = () => {
+      this.myData = JSON.parse(localStorage.getItem('bookData')) || [];
+      return this.myData;
+    }
+
+    addBook = () => {
+      const newBook = { title: title.value, author: author.value };
+      const authorRegex = /^[a-zA-Z\s]*$/;
+      // eslint-disable-next-line max-len
```

Comment:
- [ ] Please do not disable linters. Linters help us write good-quality code that conforms to a uniform standard in a team.